install/src/HieInstall.hs

```diff
@@ -111,17 +109,15 @@ defaultMain = do
       )
 
     -- cabal specific targets
-    phony "cabal-build-latest" (need ["cabal-hie-" ++ last ghcVersions])
-    phony "cabal-build"  (need ["build-data", "cabal-build-latest"])
```

Comment:
The help target continues showing the previous targets for both tools:
```
D:\ws\haskell\haskell-ide-engine>cabal v2-run install.hs --project-file install\shake.project help
Resolving dependencies...
......
Linking D:\ws\haskell\haskell-ide-engine\dist-newstyle\build\x86_64-windows\ghc-8.6.5\fake-package-0\x\script\build\script\script.exe ...
run from: cabal

Usage:
    stack install.hs <target>
    or
    cabal v2-run install.hs --project-file install/shake.project <target>

Targets:
    help                   Show help message including all targets

    build                  Build hie with the latest available GHC and the data files
    build-latest           Build hie with the latest available GHC
    build-data             Get the required data-files for `hie` (Hoogle DB)
    hie-8.4.4              Builds hie for GHC version 8.4.4
    hie-8.6.4              Builds hie for GHC version 8.6.4
    hie-8.6.5              Builds hie for GHC version 8.6.5

    stack-build            Build hie with the latest available GHC and the data files; with stack
    stack-build-latest     Build hie with the latest available GHC; with stack
    stack-build-data       Get the required data-files for `hie` (Hoogle DB); with stack
    stack-hie-8.4.2        Builds hie for GHC version 8.4.2; with stack
    stack-hie-8.4.3        Builds hie for GHC version 8.4.3; with stack
    stack-hie-8.4.4        Builds hie for GHC version 8.4.4; with stack
    stack-hie-8.6.1        Builds hie for GHC version 8.6.1; with stack
    stack-hie-8.6.2        Builds hie for GHC version 8.6.2; with stack
    stack-hie-8.6.4        Builds hie for GHC version 8.6.4; with stack
    stack-hie-8.6.5        Builds hie for GHC version 8.6.5; with stack

    cabal-ghcs             Show all GHC versions that can be installed via `cabal-build`.
    cabal-build            Build hie with the latest available GHC and the data files; with cabal
    cabal-build-latest     Build hie with the latest available GHC; with cabal
    cabal-build-data       Get the required data-files for `hie` (Hoogle DB); with cabal
    cabal-hie-8.4.4        Builds hie for GHC version 8.4.4; with cabal
    cabal-hie-8.6.4        Builds hie for GHC version 8.6.4; with cabal
    cabal-hie-8.6.5        Builds hie for GHC version 8.6.5; with cabal

    icu-macos-fix          Fixes icu related problems in MacOS

Build completed in 0.04s
```

It should be now something like:

```
Targets:
    help                   Show help message including all targets

    build                  Build hie with the latest available GHC and the data files
    build-latest           Build hie with the latest available GHC
    build-data             Get the required data-files for `hie` (Hoogle DB)
    hie-8.4.4              Builds hie for GHC version 8.4.4
    hie-8.6.4              Builds hie for GHC version 8.6.4
    hie-8.6.5              Builds hie for GHC version 8.6.5

    icu-macos-fix          Fixes icu related problems in MacOS
```